triton/uploader.go

```diff
@@ -45,6 +47,26 @@ func (s *S3Uploader) Upload(fileName, keyName string) (err error) {
 	return
 }
 
+func (s *S3Uploader) UploadBuf(r io.Reader, keyName string) (err error) {
```

Comment:
This name doesn't seem exactly right... the first argument is a Reader not a buffer.